mautic/mautic
# Not contains support for campaign's form value condition expression with notLike

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Added the same logic to the form field value condition as segments filter does for like/notLike.https://github.com/kuzmany/mautic/blob/f5fd2cf553eec037f7b6211c193f8675d2331b40/app/bundles/LeadBundle/Segment/Decorator/BaseDecorator.php#L131-L131

That means, If search value doesn't contain `%` symbol, we add it `%value%`

#### Steps to test this PR:
1. Create a form with a multiselect value with the options: "first, second, third."
2. Submit it with the first and third options selected.
3. Create a campaign with a form field value condition of "not like" for the multiselect, excluding the option "first."
4. You should be true in the campaign.
